applications/dashboard/controllers/class.messagecontroller.php

```diff
@@ -201,6 +219,7 @@ protected function setEnabled($messageID, $enabled) {
         } else {
             $this->informMessage(sprintf(t('%s disabled.'), t('Message')));
         }
+        Gdn::cache()->remove('Messages');
```

Comment:
It looks like this might already be in the message model as [`MessageModel::messages(null)`](https://github.com/vanilla/vanilla/blob/5286ffe89f4b4b03a82469e77f96e2c517923912/applications/dashboard/models/class.messagemodel.php#L253) (used in `MessageModel::delete`).